www/~/%username/emails/modify.json.spt

```diff
@@ -46,7 +47,10 @@ if action in ('add-email', 'resend', 'start-verification'):
         raise Response(400)
 
     participant.start_email_verification(address, *packages)
-    msg = _("Check your inbox for a verification link.")
+    if show_address_in_message:
```

Comment:
It's safe to show when the user is in control, but better not to for e.g. verifications, where a third party can initiate a message to any address.